cmsdb/campaigns/run3_2022_postEE_nano_v12/higgs.py

```diff
@@ -944,20 +944,6 @@
         ),
     ),
 )
-cpn.add_dataset(
-    name="wmh_hzz4l_powheg",
-    id=14799041,
-    processes=[procs.wmh_hzz4l],
-    info=dict(
-        nominal=DatasetInfo(
-            keys=[
-                "/WminusH_Hto2Zto4L_M-125p5_TuneCP5_13p6TeV_powheg2-minlo-HWJ-JHUGenV752-pythia8/Run3Summer22EENanoAODv12-130X_mcRun3_2022_realistic_postEE_v6-v2/NANOAODSIM",  # noqa
-            ],
-            n_files=37,
-            n_events=1730541,
-        ),
-    ),
-)
```

Comment:
What is the reason for deleting this dataset? It is marked as valid in DAS...

Reply:
this is generated with m_H=125.5GeV. There are lots of samples with mass variations and we do not consider any of them